### DISCUSSION TITLE
Serious Fraud on Google Extensions Site: Someone else has uploaded the extension???

### DISCUSSION THREAD

**Discussion by AnxiousPleaseHelp** [elementdavv/internet_archive_downloader#62] [General]
Hello,

Someone is trying to claim they're our dear leader Elementdavv!!!

There appears to be a fraud on the Google Web Store.

I looked for discussion about this and it seems like nobody here is aware?

I thought this was a serious issue because this person uploaded the extension very recently... just after the original and genuine Internet Archive Downloader was taken down from Google's Web Store.

Link: https://chromewebstore.google.com/detail/internet-archive-download/kgkalmdehabmgbmpnkppgcnnlgeojbkl

**Does anyone know which version of Elementdavv's Internet Archive Downloader is the best and most reliable??? Firefox or Opera or Edge???**

**Comment by AnxiousPleaseHelp** [comment]
I also just noticed his reviews which all seem positive, but nobody knows that he stole the name and idea of the extension from the superman we call Elementdavv!

https://chromewebstore.google.com/detail/internet-archive-download/kgkalmdehabmgbmpnkppgcnnlgeojbkl/reviews

**Comment by AnxiousPleaseHelp** [comment]
I thought this would get a lot of attention??? 😭 

I tried contacting the author of this App but their support site https://saltedbyte.com/ is offline... 😢 😢 😢 

I'm going to install this on my Chrome Beta Install on my Win10PC  and do some testing on it.

**Reply by elementdavv** [reply]
> I thought this would get a lot of attention??? 😭
> 
> I tried contacting the author of this App but their support site https://saltedbyte.com/ is offline... 😢 😢 😢
> 
> I'm going to install this on my Chrome Beta Install on my Win10PC and do some testing on it.

It is a different function with the same name. Do not install it.

**Comment by elementdavv** [comment]
> 
> Someone is trying to claim they're our dear leader Elementdavv!!!
> 
> There appears to be a fraud on the Google Web Store.
> 
> I looked for discussion about this and it seems like nobody here is aware?
> 
> I thought this was a serious issue because this person uploaded the extension very recently... just after the original and genuine Internet Archive Downloader was taken down from Google's Web Store.
> 
> Link: https://chromewebstore.google.com/detail/internet-archive-download/kgkalmdehabmgbmpnkppgcnnlgeojbkl
> 

This extension has the same name but with different function. It is not fraud.
There is a fraud extension I found last month which copied the version 0.8.0 but it seems disappeared now? 

> **Does anyone know which version of Elementdavv's Internet Archive Downloader is the best and most reliable??? Firefox or Opera or Edge???**

The function of extension is same for different browsers if the version is identical. If you stress on the stability, I will suggest Chrome and Edge versions, because Firefox lacks some indispensable functions which I have to employ some tricks.

**Reply by elementdavv** [reply]
> There is a fraud extension I found last month which copied the version 0.8.0 but it seems disappeared now?
> 

The fraud appeared again by the same person: 
https://chromewebstore.google.com/detail/internet-archive-download/bibnckpdfoafofnkmcdoacackddbhcfm

**Reply by AnxiousPleaseHelp** [reply]
> > There is a fraud extension I found last month which copied the version 0.8.0 but it seems disappeared now?
> 
> The fraud appeared again by the same person: https://chromewebstore.google.com/detail/internet-archive-download/bibnckpdfoafofnkmcdoacackddbhcfm

Wow, that's unbelievable....

**Reply by AnxiousPleaseHelp** [reply]
> > There is a fraud extension I found last month which copied the version 0.8.0 but it seems disappeared now?
> 
> The fraud appeared again by the same person: https://chromewebstore.google.com/detail/internet-archive-download/bibnckpdfoafofnkmcdoacackddbhcfm



> > There is a fraud extension I found last month which copied the version 0.8.0 but it seems disappeared now?
> 
> The fraud appeared again by the same person: https://chromewebstore.google.com/detail/internet-archive-download/bibnckpdfoafofnkmcdoacackddbhcfm



Wow I just re-checked the link: https://chromewebstore.google.com/detail/internet-archive-download/bibnckpdfoafofnkmcdoacackddbhcfm

He's now even taken the green extension icon you were using... I tried adding it to this post but I don't know if it worked?

How can he get away with this???
![logo128](https://github.com/user-attachments/assets/9c7d1844-0bdb-4fa1-a4b3-6be3d973946b)

**Reply by elementdavv** [reply]
> Wow I just re-checked the link: https://chromewebstore.google.com/detail/internet-archive-download/bibnckpdfoafofnkmcdoacackddbhcfm
> 
> He's now even taken the green extension icon you were using... I tried adding it to this post but I don't know if it worked?
> 
> How can he get away with this???

he copied everything, only changed version number.

**Reply by AnxiousPleaseHelp** [reply]
> > Wow I just re-checked the link: https://chromewebstore.google.com/detail/internet-archive-download/bibnckpdfoafofnkmcdoacackddbhcfm
> > He's now even taken the green extension icon you were using... I tried adding it to this post but I don't know if it worked?
> > How can he get away with this???
> 
> he copied everything, only changed version number.

That sucks bro! I can't imagine how hard you worked then to see some guy just copy your idea and claim the idea as his own... :( :( :(

Update:
I checked and there are now 3 copycats...:
https://chromewebstore.google.com/detail/internet-archive-download/bibnckpdfoafofnkmcdoacackddbhcfm

https://chromewebstore.google.com/detail/internet-archive-download/bibnckpdfoafofnkmcdoacackddbhcfm

https://chromewebstore.google.com/detail/internet-archive-download/nimnhhcainjoacphlmhbkodofenjgobh

**Would you recommend we use the Firefox and Edge extensions instead???**

**Reply by elementdavv** [reply]
> **Would you recommend we use the Firefox and Edge extensions instead???**

No. Chrome and Edge extensions are the same.

**Reply by elementdavv** [reply]
> Update: I checked and there are now 3 copycats...: https://chromewebstore.google.com/detail/internet-archive-download/bibnckpdfoafofnkmcdoacackddbhcfm
> 
> https://chromewebstore.google.com/detail/internet-archive-download/bibnckpdfoafofnkmcdoacackddbhcfm
> 
> https://chromewebstore.google.com/detail/internet-archive-download/nimnhhcainjoacphlmhbkodofenjgobh

One month ago I report these two links to Google for infringement. They replied and said they would not take any actions. Just now I checked again the links and found the first one had changed to another function. I don't know why.

**Reply by AnxiousPleaseHelp** [reply]
> > Update: I checked and there are now 3 copycats...: https://chromewebstore.google.com/detail/internet-archive-download/bibnckpdfoafofnkmcdoacackddbhcfm
> > https://chromewebstore.google.com/detail/internet-archive-download/bibnckpdfoafofnkmcdoacackddbhcfm
> > https://chromewebstore.google.com/detail/internet-archive-download/nimnhhcainjoacphlmhbkodofenjgobh
> 
> One month ago I report these two links to Google for infringement. They replied and said they would not take any actions. Just now I checked again the links and found the first one had changed to another function. I don't know why.

That really sucks bro... People are taking your work and using it as their own!

**Comment by satonotdead** [comment]
It's expected to happen, corporations always play shady.

**Reply by AnxiousPleaseHelp** [reply]
> It's expected to happen, corporations always play shady.

That's not what's happening here though. These fraudsters are just taking Elements work and publishing it as their own. It's straight out plagiarism...

> > Update: I checked and there are now 3 copycats...: https://chromewebstore.google.com/detail/internet-archive-download/bibnckpdfoafofnkmcdoacackddbhcfm
> > https://chromewebstore.google.com/detail/internet-archive-download/bibnckpdfoafofnkmcdoacackddbhcfm
> > https://chromewebstore.google.com/detail/internet-archive-download/nimnhhcainjoacphlmhbkodofenjgobh
> 
> One month ago I report these two links to Google for infringement. They replied and said they would not take any actions. Just now I checked again the links and found the first one had changed to another function. I don't know why.

**Just so it's clear, the Edge Addons and Firefox Extensions are the best versions of Internet Archive Downloader to use???**

**Reply by elementdavv** [reply]
> **Just so it's clear, the Edge Addons and Firefox Extensions are the best versions of Internet Archive Downloader to use???**

Edge Addons and Firefox Extensions have their official repositories and seems more officially and safe. Chrome Extensions can only download and install from github releases. That is the only difference. Chrome Extensions are as safe as Edge because their codes are all identical. Besides, when new releases published, Edge Extensions always update slowly because of their slow review process.

**Reply by AnxiousPleaseHelp** [reply]
> > **Just so it's clear, the Edge Addons and Firefox Extensions are the best versions of Internet Archive Downloader to use???**
> 
> Edge Addons and Firefox Extensions have their official repositories and seems more officially and safe. Chrome Extensions can only download and install from github releases. That is the only difference. Chrome Extensions are as safe as Edge because their codes are all identical. Besides, when new releases published, Edge Extensions always update slowly because of their slow review process.

Okay I had no idea that Firefox was lacking some functions that you needed to add some tricks to get working.

It seems like the Edge extension is the best version to use because it's hosted on the official Edge Extensions page: https://microsoftedge.microsoft.com/addons/detail/internet-archive-download/cnpoedgimjaecinmgfnfhfmcpcngeeje?hl=en-GB

I keep worrying that I'm downloading an ebook from the Internet Archive that doesn't contain all the all pages/text as the other versions for the other extensions like Firefox, Chrome etc.

So the Edge extension is the best version to use, right? I just want to make sure. I'll try to use Edge Canary to download some ebooks using Internet Archive Downloader.

**Reply by elementdavv** [reply]
> I keep worrying that I'm downloading an ebook from the Internet Archive that doesn't contain all the all pages/text as the other versions for the other extensions like Firefox, Chrome etc.

In coding, downloads success only if every page of the book successfully retrieved. If any page failed(even the text of the page, if you choose to include it), the whole download job failed then the book file will be deleted from your disk. So you can determine that the downloaded file includes all pages. It is nothing to do with Edge/Chrome/Firefox extension.
